customizer/samples/microsoft-teams/Share to Teams for CNX 8.0/README.md

```diff
@@ -0,0 +1,14 @@
+# Enabling MS Teams within Share 8.0
```

Comment:
Minor issue - could you change this to "Enabling Share to Microsoft Teams for Connections 8.0" ?

Reply:
README updated